### ISSUE TITLE
Backward compatibility

### ISSUE THREAD

**Issue by Xpjay** [JoshHalvorson/calendar-date-range-picker#23]
Hey,

Not sure if it's useful but maybe min sdk version 26 is too high.
For a better compatibility maybe it would be better to add code for min sdk version 19 ?

More compatibility = more people who use your library (indeed means more problems haha)

**Comment by JoshHalvorson** [comment]
Yeah would definitely be a good idea! Hopefully its not too tough lol. Thank you!
